playbooks/openshift-checks/health.yml

```diff
@@ -0,0 +1,15 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- name: Run OpenShift health checks
+  hosts: OSEv3
+  # Temporarily reverting to OSEv3 until group standardization is complete
```

Comment:
Well git history will be confusing as we're moving things around... but you're right, this is more a commit message.